barretenberg/cpp/src/barretenberg/stdlib/primitives/group/cycle_group.cpp

```diff
@@ -1185,6 +1198,11 @@ cycle_group<Composer> cycle_group<Composer>::batch_mul(const std::vector<cycle_s
     size_t num_bits = 0;
     for (auto& s : scalars) {
         num_bits = std::max(num_bits, s.num_bits());
+
+        // Note: is this the best place to put `validate_is_in_field`? Should it not be part of the constructor?
```

Comment:
Yeah I think this should be a aprt of the constructor, so that each cycle_scalar will by construction be valid